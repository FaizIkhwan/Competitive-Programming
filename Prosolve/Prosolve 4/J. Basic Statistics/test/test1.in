12 12 -63 23 -91 -49 -65 -32 12
