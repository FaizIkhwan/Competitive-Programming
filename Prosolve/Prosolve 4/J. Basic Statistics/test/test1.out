Mean : -26.78
Median : -91.00
